frontend/admin/src/js/components/user/GeneralInformationTab.tsx

```diff
@@ -0,0 +1,676 @@
+            defaultMessage: "Person with disability",
+            description:
+              "Text on view-user page that the user has a disability",
+          })}
+        </div>
+      )}
+      {user.isVisibleMinority && (
+        <div data-h2-padding="b(top-bottom, xxs)">
+          <CheckIcon style={{ width: "1rem" }} />
+          {"  "}
+          {intl.formatMessage({
+            defaultMessage: "Visible minority",
+            description:
+              "Text on view-user page that the user is part of a visible minority",
+          })}
+        </div>
+      )}
+      {user.isWoman && (
+        <div data-h2-padding="b(top-bottom, xxs)">
+          <CheckIcon style={{ width: "1rem" }} />
+          {"  "}
+          {intl.formatMessage({
+            defaultMessage: "Woman",
+            description: "Text on view-user page that the user is a woman",
+          })}
+        </div>
+      )}
+    </div>
+  );
+};
+
+const GeneralInformationTab: React.FC<SectionWithPoolsProps> = ({
+  user,
+  pools,
+}) => {
+  const intl = useIntl();
+
+  const items = [
+    {
+      id: "about",
+      title: intl.formatMessage({
+        defaultMessage: "About",
+        description: "Title of the 'About' section of the view-user page",
+      }),
+      titleIcon: <UserIcon style={{ width: "2rem" }} />,
+      content: <AboutSection user={user} />,
+    },
+    {
+      id: "candidate-status",
+      title: intl.formatMessage({
+        defaultMessage: "Candidate status",
+        description:
+          "Title of the 'Candidate status' section of the view-user page",
+      }),
+      titleIcon: <CalculatorIcon style={{ width: "2rem" }} />,
+      content: <CandidateStatusSection user={user} pools={pools} />,
+    },
+    {
+      id: "notes",
+      title: intl.formatMessage({
+        defaultMessage: "Notes",
+        description: "Title of the 'Notes' section of the view-user page",
+      }),
+      titleIcon: <PencilAltIcon style={{ width: "2rem" }} />,
+      content: <NotesSection user={user} />,
+    },
+    {
+      id: "employment-equity",
+      title: intl.formatMessage({
+        defaultMessage: "Employment equity",
+        description:
+          "Title of the 'Employment equity' section of the view-user page",
+      }),
+      titleIcon: <InformationCircleIcon style={{ width: "2rem" }} />,
+      content: <EmploymentEquitySection user={user} />,
+    },
+  ];
+
+  return (
+    <TableOfContents.Wrapper>
+      <TableOfContents.Navigation>
+        {items.map((item) => (
+          <TableOfContents.AnchorLink key={item.id} id={item.id}>
+            {item.title}
+          </TableOfContents.AnchorLink>
+        ))}
+      </TableOfContents.Navigation>
+      <TableOfContents.Content>
+        {items.map((item) => (
+          <TableOfContents.Section key={item.id} id={item.id}>
+            <TableOfContents.Heading>
+              {item.titleIcon} {item.title}
+            </TableOfContents.Heading>
```

Comment:
You can actually just pass the icon directly to the `<TableofContents.Heading>` component as a prop and it will apply styles for you.

```tsx
<TableOfContents.Heading icon={UserIcon}>
  {item.title}
</TableOfContents.Heading>
```